libs/pages/settings/src/lib/feature/page-organization-general-feature/page-organization-general-feature.tsx

```diff
@@ -1,7 +1,77 @@
+import { Organization } from 'qovery-typescript-axios'
+import { useEffect, useState } from 'react'
+import { FieldValues, FormProvider, useForm } from 'react-hook-form'
+import { useDispatch, useSelector } from 'react-redux'
+import { useParams } from 'react-router-dom'
+import { editOrganization, selectOrganizationById } from '@qovery/domains/organization'
+import { useDocumentTitle } from '@qovery/shared/utils'
+import { AppDispatch, RootState } from '@qovery/store/data'
 import PageOrganizationGeneral from '../../ui/page-organization-general/page-organization-general'
 
+export const handleSubmit = (data: FieldValues, organization: Organization) => {
```

Comment:
why you don't use the spread operator function here?
```
return {
   ...organization,
   logoUrl: data.logo_url,
   name: data.name,
   etc
}
```

It does the same thing but just quicker to write and read